Makefile

```diff
@@ -326,6 +326,44 @@ fix-ownership:
 	docker-compose exec php-fpm bash -c \
 		"[[ -e public ]] && find public ! -user ${APP_USER} -exec chown -f ${APP_USER} {} \;"
 
+.ONESHELL:
+check-content-before-install: envcheck
+	@export $$(sed -e '/^\#/d' .env | xargs)
+	@export $$(sed -e '/^\#/d' app.env | xargs)
+	content_exists=false
+	echo "Checking content for project $${COMPOSE_PROJECT_NAME} ..."
+	if [[ $$(ls persistence 2>/dev/null | wc -l) -gt 0 ]]; then
```

Comment:
You could also do 
```bash
if [ -z "$(ls -A /path/to/dir)" ]; then
```
here and in the following checks. Maybe a bit more compact.